crawler/helpers/cherrypicker_test_data.py

```diff
@@ -0,0 +1,157 @@
+import csv
+import logging
+import os
+import random
+from datetime import datetime
+
+import requests
+
+from crawler.types import Config
+
+logger = logging.getLogger(__name__)
+
+
+def make_coords_list():
+    for column in "ABCDEFGH":
+        for row in range(1, 13):
+            yield "%s%02d" % (column, row)
```

Comment:
I think f-string are clearer: `f"{column}{row:02d}"`.

Reply:
I only did it this way because I didn't know how to apply a number formatter with an f-string.  There's still scope for percentages in some scenarios like where you define the string in a constant and insert values into it later, but that doesn't apply here.  I'll change it.